docs/Using Fleet/Vulnerability-Processing.md

```diff
@@ -19,91 +13,21 @@ Vulnerable software can have one or more vulnerabilities (CVEs).
-for which we use the release notes published by Microsoft. For Linux hosts,
-CVEs are detected using the official OVAL definitions maintained by the different publishers (Canonical, Red Hat etc.).
-
-### Windows/MacOS hosts using the NVD dataset
-
-First, Fleet retrieves the installed software for each host using osquery queries. Then, Fleet
-translates each installed software into [Common Platform Enumeration (CPE)](https://en.wikipedia.org/wiki/Common_Platform_Enumeration) names.
-
-Then, using the CPEs, Fleet searches the list of [Common Vulnerabilities and Exposure
-(CVE)](https://en.wikipedia.org/wiki/Common_Vulnerabilities_and_Exposures) identifiers listed in the NVD
-to detect the CVEs that match the defined CPEs.
-
-If matches are found, they are exposed on each host's
-**Host details** page and on the **Home** page in the Fleet UI. The CVEs are also exposed in the
-`fleetctl get software` command and the `GET api/v1/fleet/hosts/{id}` and `GET
-api/v1/fleet/software` API routes.
-
-Vulnerability processing happens on the Fleet instance and not on the host machine. Because of this,
-detected vulnerabilities cannot be used in the same way you would use an osquery query (e.g. you wouldn't
-be able write a query to retrieve all CVEs).
-
-### MacOffice release notes
+the software in question. CVEs are detected using:
+- **macOS and Windows**: National Vulnerability Database (NVD) except for Mac Office applications, for which we use the release notes published by Microsoft.
+- **Linux**: OVAL definitions maintained by the different publishers (Canonical, Red Hat etc.).
 
-We periodically parse the Mac Office release notes posted at
-<https://learn.microsoft.com/en-us/officeupdates/release-notes-office-for-mac> and for each release
-note we extract the version
-information and what vulnerabilities were patched with the release, we then examine all installed
-macOS apps and if an Office app is found we compare its version with the release notes metadata
-and report back any vulnerabilities to which the software is susceptible.
-
-### Linux hosts
-
-First, we determine what Linux distributions are part of your fleet (keep in mind that there will
-be a small delay between the time a new Linux hosts is added and the time the host is 'detected'). We then
-use
-that information to determine what OVAL definitions need to be downloaded and parsed - you can find
-a list of all the OVAL definitions we use
-[here](https://github.com/fleetdm/nvd/blob/master/oval_sources.json). OVAL definitions will be
-refreshed on a daily basis.
-
-Finally, we look at the software inventory of each host and execute the assertions contained in the
-corresponding OVAL file - any match is reported using the same channels as with Windows/Mac OS vulnerabilities
+[Learn more](https://github.com/fleetdm/fleet/blob/main/docs/Contributing/Vulnerability-processing.md) about performance and how it works for different platforms.
```

Comment:
```suggestion
[Learn more](https://github.com/fleetdm/fleet/blob/main/docs/Contributing/Vulnerability-processing.md) about how it works for different platforms.
```

Reply:
@noahtalerman I was thinking if someone tries to search for performance would get to this link which navigates to contributor docs. Does it make sense to add an additional sentence mentioning performance?

Reply:
Eh, I doubt someone's looking for performance. I think we cut it for sake of less words.